nearcore/src/config.rs

```diff
@@ -431,6 +431,9 @@ pub struct Config {
     pub view_client_throttle_period: Duration,
     #[serde(default = "default_trie_viewer_state_size_limit")]
     pub trie_viewer_state_size_limit: Option<u64>,
+    /// If set, overrides value in genesis configuration.
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Hmm, so intention is to allow keeping backward and forward compatibility with serilized data if option is None?

Reply:
Mostly yes. Technically it’s probably not necessary (since IIUC unknown fields are ignored and missing fields get None value if they are Option type) but I didn’t actually verify that. There are other Option fields which are skipped if None so precedent is there.